package-lock.json

```diff
@@ -58686,7 +58742,7 @@
 		},
 		"packages/a11y": {
 			"name": "@wordpress/a11y",
-			"version": "3.38.0",
+			"version": "3.39.0",
```

Comment:
I have no idea why I had to rebase and push these changes? I didn't change the versions of these packages myself.

Reply:
I fixed it in https://github.com/WordPress/gutenberg/commit/3e7cdc79846226f2fd993c202457466a92acaea7. The long story short:
- npm publishing failed last week for the branch using npm 6/node 14
- I executed it manually today, and I had to manually cherry-pick changes to `package.json` and `package-lock.json`.
- It turns out that the lock files have a different format with npm 6 vs npm 8 so I didn't catch that immediately because cherry-picking went well ...